cli/src/commands/dbs/list.ts

```diff
@@ -1,13 +1,13 @@
 import { Flags } from '@oclif/core';
 import { BaseCommand } from '../../base.js';
-import { parseDatabaseURL } from '../../defaults.js';
 
 export default class DatabasesList extends BaseCommand {
   static description = 'List your databases';
 
   static examples = [];
 
   static flags = {
+    json: Flags.boolean({}),
```

Comment:
Does this make sense?

```suggestion
    ...this.commonFlags,
```

Reply:
Yep. I was testing with this file and forgot to update it 😅